api/database/database.js

```diff
@@ -4,13 +4,15 @@ module.exports = {
     password: process.env.DEV_DB_PASSWORD,
     database: process.env.DEV_DB_NAME,
     host: process.env.DEV_DB_HOSTNAME,
+    logging: false,
```

Comment:
Is this a sequelize specific thing?

Reply:
yeah, it's so it doesn't print every single request

Reply:
Oh, can we keep it in there so we can pipe logs eventually

Reply:
I left in for prod